extension/test/data_test.ts

```diff
@@ -134,6 +140,48 @@ describe('data.ts', function () {
           '<word-1>\t<pronunciation-1>\t<definition-1>; <definition-2>; <definition-3>\n<word-2>\t<definition-1>; <definition-2>\n'
         );
       });
+
+      describe('when data entry is an invalid format', function () {
```

Comment:
We might restructure the data types for the dict entries in the future so that the values are parsed and checked before this function is even called, so we might be able to delete these tests in the future.

I'll leave these tests in for now just so the current behavior is documented and so we won't have to deal with code coverage problems in `makeText` for now.